lib/membrane_h264_plugin/parser.ex

```diff
@@ -589,32 +595,17 @@ defmodule Membrane.H264.Parser do
     nalus =
       nalus
       |> Enum.with_index()
-      |> Enum.map_reduce(0, fn {nalu, i}, nalu_start ->
-        metadata = %{
-          metadata: %{
-            h264: %{
-              type: nalu.type
-            }
-          }
-        }
-
-        metadata =
-          if i == length(nalus) - 1 do
-            put_in(metadata, [:metadata, :h264, :end_access_unit], true)
-          else
-            metadata
-          end
-
-        metadata =
-          if i == 0 do
-            put_in(metadata, [:metadata, :h264, :new_access_unit], %{key_frame?: is_keyframe?})
-          else
-            metadata
-          end
-
-        {metadata, nalu_start + byte_size(nalu.payload)}
```

Comment:
I am slightly surprised that `nalu_start` is not used anywhere :D

Reply:
It was used to generate `prefixed_poslen` and `unprefixed_poslen`, but i didn't remove it then